### ISSUE TITLE
One question about pretrained model of vgg19

### ISSUE THREAD

**Issue by xuzhongyou** [pytorch/vision#1302]
I used the model vgg19 without norm or dropout and load  official state_dict. When i test one image, it returns the different label。However, when i set model.eval(), it return one result. I want to know the explaination hehind it.
![image](https://user-images.githubusercontent.com/16487416/64409304-39e0b400-d0bb-11e9-9599-e07674dafe6c.png)

**Comment by xuzhongyou** [comment]
i find the drpout in classifier,cry,cry
